### ISSUE TITLE
[Vue warn]: Invalid prop: type check failed for prop "style"

### ISSUE THREAD

**Issue by ijkml** [nuxt-modules/mdc#54]
I'm running into this warning (both terminal and browser console) whenever I go to a page that has ProseCode.

The terminal warning is:
`[Vue warn]: Invalid prop: type check failed for prop "style". Expected String with value "[object Object]", got Object`

One for each code block on the page. The browser warning is much longer (provided below).

This is a simple project with no external deps. I installed afresh several times, didn't help. Weird, I can't reproduce this on Stackblitz, so it's probably a me-problem.
Repo: https://github.com/ijkml/content-app
Stackblitz: https://stackblitz.com/edit/github-ybzbbs

Nuxi Info:
```
- Operating System: `Linux`
- Node Version:     `v18.17.1`
- Nuxt Version:     `3.7.4`
- CLI Version:      `3.9.0`
- Nitro Version:    `2.6.3`
- Package Manager:  `pnpm@8.7.5`
- Builder:          `-`
- User Config:      `modules`, `content`
- Runtime Modules:  `@nuxt/content@2.8.4`
- Build Modules:    `-`
```

**Comment by ijkml** [comment]
```
[Vue warn]: Invalid prop: type check failed for prop "style". Expected String with value "[object Object]", got Object  
  at <ProsePre class="language-typescript shiki shiki-themes vitesse-light" code="codestring..." filename="filename"  ... > 
  at <AsyncComponentWrapper class="language-typescript shiki shiki-themes vitesse-light" code="codestring..." filename="filename"  ... > 
  at <MDCRenderer body= 
Object { type: "root", children: (12) […], toc: {…} }
 data= 
Object { _path: "/", _dir: "", _draft: false, _partial: false, _locale: "", title: "Nuxt Content", description: "This page corresponds to the / route of your website. You can delete it or create another file in the content/ directory.", _type: "markdown", _id: "content:index.md", _source: "content", … }
 tag="div"  ... > 
  at <ContentRendererMarkdown value= 
Object { _path: "/", _dir: "", _draft: false, _partial: false, _locale: "", title: "Nuxt Content", description: "This page corresponds to the / route of your website. You can delete it or create another file in the content/ directory.", body: {…}, _type: "markdown", _id: "content:index.md", … }
 excerpt=false tag="div" > 
  at <ContentRenderer value= 
Object { _path: "/", _dir: "", _draft: false, _partial: false, _locale: "", title: "Nuxt Content", description: "This page corresponds to the / route of your website. You can delete it or create another file in the content/ directory.", body: {…}, _type: "markdown", _id: "content:index.md", … }
 excerpt=false tag="div" > 
  at <ContentQuery path="/" find="one" > 
  at <ContentDoc> 
  at <[...slug] onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouteProvider key="/" vnode= 
Object { __v_isVNode: true, __v_skip: true, type: {…}, props: {…}, key: null, ref: {…}, scopeId: null, slotScopeIds: null, children: null, component: null, … }
 route= 
Object { fullPath: "/", hash: "", query: {}, name: "slug", path: "/", params: {…}, matched: (1) […], meta: Proxy, redirectedFrom: undefined, href: "/" }
  ... > 
  at <RouterView name=undefined route=undefined > 
  at <NuxtPage> 
  at <App key=3 > 
  at <NuxtRoot> deps:1449:13
```

**Comment by farnabaz** [comment]
This should be fixed in @nuxt/content@2.8.5.

I'm closing this issue, feel free to ping or create new one if issue proceeds.

**Comment by ijkml** [comment]
Can confirm, fixed, thank you!
